app/lib/frontend/templates/views/pkg/score_tab.dart

```diff
@@ -53,21 +54,31 @@ d.Node scoreTabNode({
           ),
           d.text(' for its analysis.'),
         ],
-      ),
-    if (showReport)
-      d.p(
-        classes: ['analysis-info'],
-        children: [
-          d.text('We analyzed this package '),
-          if (card.panaReport?.timestamp != null)
-            d.xAgoTimestamp(card.panaReport!.timestamp!, datePrefix: 'on'),
-          d.text(', '
-              'and awarded it ${report.grantedPoints} '
-              'pub points (of a possible ${report.maxPoints}):'),
-        ],
-      ),
-    if (report != null) _reportNode(report),
-    if (toolEnvInfo != null) toolEnvInfo,
+      )
+    else
+      d.fragment([
+        d.p(
+          classes: ['analysis-info'],
+          children: [
+            d.text('We analyzed this package '),
+            if (card.panaReport?.timestamp != null)
+              d.xAgoTimestamp(card.panaReport!.timestamp!, datePrefix: 'on'),
+            d.text(', '
+                'and awarded it ${report!.grantedPoints} '
+                'pub points (of a possible ${report.maxPoints}):'),
+          ],
+        ),
+        _reportNode(report),
+        if (toolEnvInfo != null) toolEnvInfo,
+        d.p(
+          classes: ['analysis-info'],
+          children: [
+            d.text('You can check the '),
+            d.a(href: 'score/log.txt', text: 'analysis log'),
+            d.text(' for more details.'),
```

Comment:
nit:

```suggestion
            d.text('Check '),
            d.a(href: 'log.txt', text: 'analysis log'),
            d.text(' for details.'),
```

I think it says that same, just with fewer words, right?